node/malus/integrationtests/0001-dispute-valid-block.feature

```diff
@@ -0,0 +1,28 @@
+Description: Disputes
+Network: ./0001-dispute-valid-block.toml
+Creds: config.gcloud
+
+
+alice: is up
+bob: is up
+charlie: is up
+david is up
+alice: reports node_roles is 4
+bob: reports node_roles is 4
+alice: reports sub_libp2p_is_major_syncing is 0
+alice: reports block height is at least 2 within 15 seconds
+alice: reports peers count is at least 2
+bob: reports block height is at least 2
+bob: reports peers count is at least 2
+charlie: reports block height is at least 2
+charlie: reports peers count is at least 2
+alice: reports parachain_candidate_disputes_total is at least 1 within 201 seconds
+bob: reports parachain_candidate_disputes_total is at least 1 within 5 seconds
+charlie: reports parachain_candidate_disputes_total is at least 1  within 5 seconds
+alice: reports parachain_candidate_dispute_votes{validity="valid"} is at least 1
+bob: reports parachain_candidate_dispute_votes{validity="valid"} is at least 2
+charlie: reports parachain_candidate_dispute_votes{validity="valid"} is at least 2
+alice: reports parachain_candidate_dispute_concluded{validity="valid"} is at least 1
+alice: reports parachain_candidate_dispute_concluded{validity="invalid"} is 0
```

Comment:
Hi @drahnr / @ordian, I add a larger timeout (using in within 30 seconds) but this metrics doesn't show at all. The rest of the test are ok with the `in within` logic.  Could be the case that we don't report any metric is there is any invalid block?

Reply:
Hey, it's known that prometheus doesn't report 0 if a metric has never been reported, we need to special case that. cf. https://github.com/paritytech/gurke/pull/143

Reply:
Hi @ordian thanks for the help, I will also implement this special case in zombie-net.
Thanks!